.vscode/launch.json

```diff
@@ -332,7 +332,12 @@
             "type": "pickString",
             "id": "mbedApp",
             "description": "What mbed application do you want to use?",
-            "options": ["shell", "lock-app", "lighting-app", "all-clusters-app"],
+            "options": [
+                "shell",
+                "lock-app",
+                "lighting-app",
+                "all-clusters-app"
+            ],
```

Comment:
I guess this change comes from VScode's general auto-formatting tool.
it is only present when you open our repository in vscode which is not opened in the container.
Because you didn't add anything new here, please revert this change.